### PR TITLE
Unique VirtualMachine DNS entries

### PR DESCRIPTION
**Pull request by rmohr**
Documentation for https://github.com/kubevirt/kubevirt/pull/784

### CODE REVIEW

File: workloads/virtual-machines/expose-service.md [hunk 1]
```diff
@@ -1,9 +1,94 @@
...
+
+In order to create unique DNS entries per VirtualMachine, it is possible to set
+`spec.hostname` and `spec.subdomain`. If a subdomain is set and a headless
+service with a name, matching the subdomain, exists, kube-dns will create
+unique DNS entries for every VirtualMachine which matches the selector of the
+service. Have a look at the [DNS for Servies and Pods
+documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pods-hostname-and-subdomain-fields)
+for additional information.
+
+The following example consists of a VirtualMchine and a headless Service witch
```
Comment (davidvossel) [inline workloads/virtual-machines/expose-service.md]: s/witch/which

File: workloads/virtual-machines/expose-service.md [hunk 2]
```diff
@@ -1,9 +1,94 @@
 # Expose Service
```
Comment (davidvossel) [inline workloads/virtual-machines/expose-service.md]: should we make networking section where DNS and Expose Service are separate sections under Networking?  There are other aspects of networking that need to be documented as well that probably belong in a top level networking section.

Comment (rmohr) [reply workloads/virtual-machines/expose-service.md]: Yes, makes sense.

File: workloads/virtual-machines/expose-service.md (hunk 1 above)
Comment (stu-gott) [inline workloads/virtual-machines/expose-service.md]: s/VirtualMchine/VirtualMachine/

File: workloads/virtual-machines/expose-service.md (hunk 2 above)
Comment (rmohr) [reply workloads/virtual-machines/expose-service.md]: Done

**Comment by rmohr** [comment]
@davidvossel could you have another look?

**Pull request merged**